site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2989,17 +2989,60 @@ public function getSession($session_id) {
     }
 
     /**
-     * @todo: write phpdoc
+     * Fetch the session creation timestamp and other information for all active sessions of the given user
+     * @param string $user_id
+     *
+     * @return array
+     */
+    public function getSessionsInfoByUser($user_id) {
+        $this->submitty_db->query("SELECT session_id, csrf_token, session_created, browser_name, browser_version, platform FROM sessions WHERE user_id=? ORDER BY session_created", [$user_id]);
```

Comment:
It would be more readable to break this into several lines (see other functions in this file)